h2o-core/src/main/java/water/rapids/Merge.java

```diff
@@ -327,26 +329,140 @@ public void map(Chunk[] cs, NewChunk[] ncs) {
     }
     Key<Vec> key = Vec.newKey();
     Vec[] vecs = new Vec(key, Vec.ESPC.rowLayout(key, espc)).makeCons(numColsInResult, 0, doms, types);
-    System.out.println("took: " + (System.nanoTime() - t0) / 1e9);
+    Log.info("took: " + (System.nanoTime() - t0) / 1e9+" seconds.");
 
     System.out.print("Finally stitch together by overwriting dummies ...");
     t0 = System.nanoTime();
     Frame fr = new Frame(names, vecs);
     ChunkStitcher ff = new ChunkStitcher(chunkSizes, chunkLeftMSB, chunkRightMSB, chunkBatch);
     ff.doAll(fr);
-    System.out.println("took: " + (System.nanoTime() - t0) / 1e9);
+    Log.info("took: " + (System.nanoTime() - t0) / 1e9+" seconds");
```

Comment:
are all these time infos important to the user? maybe some of them should be just DEBUG

Reply:
I think you have a point

Reply:
I left the ones telling user what the system is doing as info but the speed info as debug.